README.md

```diff
@@ -80,16 +80,21 @@ $ ./scripts/rpc.py send_money --help
 $ ./scripts/rpc.py send_money -r bob -a 1
 
 # Deploy a smart contract
-./scripts/rpc.py deploy test_contract core/wasm/runtest/res/wasm_with_mem.wasm
+$ ./scripts/rpc.py deploy test_contract core/wasm/runtest/res/wasm_with_mem.wasm
```

Comment:
IMO it's better to remove `$` everywhere as it just messes with easy copy-pasting.
When command needs to be run as root it can be prefixed with `sudo`